src/components/App/index.tsx

```diff
@@ -24,3 +32,8 @@ export function App(_: Props) {
     </div>
   )
 }
+
+App.defaultProps = {
+  children: undefined,
+  isAnimated: true,
+}
```

Comment:
Let's prefer default params on the function directly. It's more direct and works better from a type standpoint.

Reply:
But also is rendered necessary by my next suggestion 😬